Improve search experience

DengYiping opened the pull request:
Final release.

DengYiping committed:
Experimental release

I will test it on my server soon.

DengYiping committed:
Improve search experience, add new keyboard.

Final release.

sudheesh001 commented on bot.py:
```diff
@@ -2,13 +2,14 @@
-
-bot = telebot.TeleBot(os.environ['LOKLAK_TELEGRAM_TOKEN'])
+#put your token in the line below
+bot = telebot.TeleBot("PUT YOUR TOKEN HERE")
```
refactor this into a new file called `token.py` from which do

`from token import TOKEN` where
`TOKEN = '****Your long token over here***'`

sevazhidkov commented:
Thank you, cool search method. Why did you change the method of getting token? Defining tokens in code isn't good experience.

DengYiping committed:
Add user search function

try /user:CBS

DengYiping committed:
some small changes

DengYiping commented:
I changed it to what @sudheesh001 suggested. Now you can safely merge. I also add some new function like doing /user:CBS

sudheesh001 commented on token.py:
```diff
@@ -0,0 +1 @@
+TOKEK = "YOUR TOKEN CODE IS HERE"
```
Nit: `TOKEN`

DengYiping replied:
Thanks

DengYiping committed:
Fix problem and add new features: user information

I rename the token because token cannot be imported in python 3. There
is a package called tokenize.  I add a feature to get user information.
Should be helpful.

DengYiping commented:
@sudheesh001 @sevazhidkov When I am trying to fix the compatibility with Python 3, I find out that one should not have a file named token.py in the dictionary because it is a system module. It only cause trouble in Python 3 on my laptop. However, we should not do that. Now the code is working under Python 3.

sevazhidkov commented:
@sudheesh001 Do you think that defining token in code is a good idea?

DengYiping commented:
@sevazhidkov You can safely merge right now. There is no token.py.

sudheesh001 commented:
@sevazhidkov Looks like @DengYiping has refactored the code, there's no more a definition of token in the code.

sevazhidkov commented:
@sudheesh001 I think that token should take from os environ variable.

sevazhidkov commented:
But I can fix it myself, it's not problem

sevazhidkov commented:
For example, if someone wants to host bot's source code in Github public repo, but also he wants to deploy bot in Heroku (or other cloud hosting), so he can just set env variable in dashboard of cloud hosting

sevazhidkov commented:
@DengYiping Thank you, very good work

DengYiping commented:
Not a problem. Great job.
